src/lib/const.ts

```diff
@@ -36,10 +36,26 @@ export const RUST_WASM32_WASI_TARGET = "wasm32-wasi";
 
 export const DEFAULT_MARINE_BUILD_ARGS = `--release`;
 
+export const numberProperties = [
+  "minPricePerWorkerEpoch",
+  "maxCollateralPerWorker",
+] as const;
+
+export type NumberProperty = (typeof numberProperties)[number];
+
+export const defaultNumberProperties: Record<NumberProperty, number> = {
+  maxCollateralPerWorker: 1 * 10 ** 18,
+  minPricePerWorkerEpoch: 83 * 10 ** 15,
```

Comment:
I suggest to extract these values to constant variable.
For example:
const ETH = 10 ** 18;
const MILLI_ETH = 10 ** 15;
...
{
maxCollateralPerWorker: ETH,
minPricePerWorkerEpoch: 83 * MILLI_ETH,
}